### PR TITLE
CVSL-215: Expanding the trixy conditions that alter the sentence text.

### PR DESCRIPTION
**Pull request by timharrison-moj**
5 of the additional conditions include additional text to make sentences make sense when optional values are either present or omitted. This PR deals with those by using `.includeBefore` fixed text which is prepended to the value when it is present, or omitted when the value is not present.

### CODE REVIEW

File: server/config/conditions.ts [hunk 1]
```diff
@@ -75,7 +75,7 @@ export default {
       code: '5db26ab3-9b6f-4bee-b2aa-53aa3f3be7dd',
       category: 'Residence at a specific place',
       text: 'You must reside within [INSERT REGION] while of no fixed abode, unless otherwise approved by your supervising officer.',
```
Comment (smcveigh941) [inline server/config/conditions.ts:77]: The text which shows on the screens should match what is on the resultant licence maybe?

Comment (timharrison-moj) [reply server/config/conditions.ts:77]: yes, it should. It altered the mean when selecting from drop down, so I'll update this to include `the [value] probation region`

Comment (timharrison-moj) [reply server/config/conditions.ts]: Pushed now.

**Review by smcveigh941 — APPROVED** [review]
LGTM

**Pull request merged**